thundra/plugins/log/log_plugin.py

```diff
@@ -0,0 +1,42 @@
+import uuid
+
+from thundra import utils, constants
+from thundra.plugins.log.thundra_log_handler import logs
+
+
+class LogPlugin:
+
+    data_format_version = '1.2'
```

Comment:
Can `data_format_version` be defined as constant at more common place for all plugins?